readme.md

```diff
@@ -0,0 +1,31 @@
+# Workflow CA Project
```

Comment:
For convention, rename this file to `README.md`

Reply:
I renamed it!